examples/hooks/App.tsx

```diff
@@ -57,6 +58,9 @@ export function App() {
               showMore={true}
             />
           </Panel>
+          <Panel header="Categories">
+            <Menu attribute="categories" showMore={true} />
```

Comment:
just `showMore` without `{true}` is enough no?
it's idiomatic enough in React that I don't worry about its readability.

Reply:
it seemed consistent with the other usages, can change after these are merged?